### PR TITLE
[generator] added theia-rebuild binary

### PR DESCRIPTION
**Pull request by akosyakov**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,9 +2,6 @@
   "private": true,
   "name": "parent",
   "version": "0.0.0",
-  "dependencies": {
-    "node-pty": "^0.6.10"
-  },
   "devDependencies": {
```
Comment (hexa00) [inline package.json]: why is this now removed ?

**Comment by hexa00** [comment]
What is the link between the openhandler and the theia-rebuild?

File: package.json (hunk 1 above)
Comment (akosyakov) [reply package.json]: The rebuild script will write package.json with node-pty automatically by default, so we don't need to duplicate it. Additional modules can be passed with `modules` option.

**Comment by akosyakov** [comment]
I am working on the standalone example of developing the Theia extension and they both need for it. Since the time around is bad (I have to wait for publishing), I put everything in one PR and ask Sven to review it.

File: package.json (hunk 1 above)
Comment (hexa00) [reply package.json]: OK got it thx, that's nice

**Comment by hexa00** [comment]
OK

**Pull request merged**